src/frontend/src/env/rest/coingecko.env.ts

```diff
@@ -0,0 +1,9 @@
+import { UrlSchema } from '$lib/validation/url.validation';
+
+const parsedCoingeckoApiUrl = UrlSchema.safeParse('https://pro-api.coingecko.com/api/v3/');
```

Comment:
If you want fancyness `const {success, data} = ...` 😉

Reply:
good one!